luxonis_train/config/predefined_models/segmentation_model.py

```diff
@@ -14,36 +14,61 @@
 
 @dataclass
 class SegmentationModel(BasePredefinedModel):
-    backbone: str = "MicroNet"
+    variant: str | None = None
```

Comment:
Keep backbone as parameter (same as with ClassificationModel) so it can be explicitly set.